src/device-manager/java/src/nl/Weave/DeviceManager/NetworkInfo.java

```diff
@@ -79,6 +90,15 @@ public NetworkInfo()
          */
     public byte[] ThreadNetworkKey;
 
+    /** The 16-bit Thread PAN ID, or kThreadPANId_NotSpecified 
+         */
+    public int ThreadPANId;
+
+    /** The current channel (currently [11..26]) on which the Thread
+         * network operates, or kThreadChannel_NotSpecified
+         */
+    public byte ThreadChannel;
```

Comment:
I think its not very conventional, nor very convenient for the Java programmer, to use byte as an integer type, except where memory usage is a factor (e.g. big arrays).   So I suggest changing this to int, a la ThreadPANId.